pkg/kafka/auth/scram.go

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2019 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package auth
+
+import (
+	"crypto/sha256"
+	"crypto/sha512"
+	"hash"
+
+	"github.com/Shopify/sarama"
+	"github.com/pkg/errors"
+	SCRAM "github.com/xdg/scram"
```

Comment:
"scram" is being used in the config file as a string constant for the AuthenticationConfig, I would need to change the constant name or change the name of the import of the package.

Reply:
but it's in a different file, why is it a problem?

Reply:
I believe because the constant is package scoped and so it conflicts with the standard import name of "scram". Would something like "scrampkg" be better?

Reply:
maybe scrumclient

Reply:
I believe that scrampkg may be better to define that it's a package, or something like xdgscram as it's the name of the import of the package `github.com/xdg/scram`.

I feel as though scrumclient or scramclient would be confusing as there's a struct named scramClient and scrum just seems mispelled. Those are my thoughts about those.